core/modules/dashboard/dashboard.module

```diff
@@ -0,0 +1,279 @@
+ */
+function dashboard_block_info() {
+  $blocks = array();
+
+  // Dashboard blocks.
+  $blocks['create'] = array(
+    'info' => t('Dashboard - Create content'),
+    'description' => t('Provides links to create new content'),
+    'class' => 'DashboardCreateBlock',
+    'required contexts' => array('dashboard'),
+  );
+  $blocks['menu'] = array(
+    'info' => t('Dashboard - Menus'),
+    'description' => t('Provides links to manage menus'),
+    'class' => 'DashboardMenuBlock',
+    'required contexts' => array('dashboard'),
+  );
+  $blocks['node_types'] = array(
+    'info' => t('Dashboard - Content types'),
+    'description' => t('Provides links to manage content types'),
+    'class' => 'DashboardContentTypesBlock',
+    'required contexts' => array('dashboard'),
+  );
+  $blocks['overview_content'] = array(
+    'info' => t('Dashboard - Content'),
+    'description' => t('Displays a summary of content statistics'),
+    'class' => 'DashboardOverviewContentBlock',
+    'required contexts' => array('dashboard'),
+  );
+  $blocks['overview_user'] = array(
+    'info' => t('Dashboard - User accounts'),
+    'description' => t('Displays a summary of user account statistics'),
+    'class' => 'DashboardOverviewUserBlock',
+    'required contexts' => array('dashboard'),
+  );
+  $blocks['updates'] = array(
+    'info' => t('Dashboard - Available updates'),
+    'description' => t('Displays a list of available updates for modules, themes, layouts, and Backdrop core.'),
+    'class' => 'DashboardUpdateBlock',
+    'required contexts' => array('dashboard'),
+  );
+
+  if (module_exists('taxonomy')) {
+    $blocks['taxonomy'] = array(
+      'info' => t('Dashboard - Categories'),
+      'description' => t('Provides links to manage taxonomy'),
+      'class' => 'DashboardTaxonomyBlock',
+      'required contexts' => array('dashboard'),
+    );
+  }
+
+  return $blocks;
+}
+
+/**
+ * Implements hook_autoload_info().
+ */
+function dashboard_autoload_info() {
+  return array(
+    'DashboardCreateBlock' => 'includes/block.create.inc',
+    'DashboardMenuBlock' => 'includes/block.menus.inc',
+    'DashboardContentTypesBlock' => 'includes/block.node_types.inc',
+    'DashboardOverviewContentBlock' => 'includes/block.overview_content.inc',
+    'DashboardOverviewUserBlock' => 'includes/block.overview_user.inc',
+    'DashboardTaxonomyBlock' => 'includes/block.taxonomy.inc',
+    'DashboardUpdateBlock' => 'includes/block.update.inc',
+    'DashboardLayoutContext' => 'includes/dashboard_layout_context.inc',
+  );
+}
+
+/**
+ * Implements hook_layout_context_info().
+ */
+function dashboard_layout_context_info() {
+  $info['dashboard'] = array(
+    'title' => t('Dashboard'),
+    'class' => 'DashboardLayoutContext',
+    'menu paths' => array(
+      'admin/dashboard',
+    ),
+  );
+  return $info;
+}
+
+/**
+ * Implements hook_user_presave().
+ */
+function dashboard_user_presave($account) {
+  cache_clear_all('dashboard_user_overview', 'cache');
+}
```

Comment:
We no longer have these caches, perhaps just remove?

Reply:
done.